content/docs/user-guide/gems/reference/environment/vegetation.md

```diff
@@ -0,0 +1,204 @@
+- [Vegetation Distribution Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-distribution-filter)
+- [Vegetation Shape Intersection Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-shape-intersection-filter)
+- [Vegetation Slope Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-slope-filter)
+- [Vegetation Surface Mask Depth Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-surface-mask-depth-filter)
+- [Vegetation Surface Mask Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-surface-mask-filter)
+
+## Vegetation Areas
+
+The dynamic vegetation system builds around the concept of *vegetation areas* that describe what, where, how, and if vegetation will generate on surfaces. 
+Other systems in the engine register and process Vegetation areas.
+They can examine and take action on points in the world, like placing or preventing a vegetation instance.
+
+In O3DE Editor, vegetation areas are referred to as *vegetation layers*.
+You can add vegetation behaviors to vegetation layers using components that describe, select, reject, or mutate potential vegetation instances. 
+These are called vegetation descriptor providers, selectors, filters, and modifiers respectively. 
+Each has an interface that extends the system with new features. 
+The Vegetation Gem contains several versions of each type that each has a unique purpose and effect on generated vegetation instances. 
+These components expose controls that allow rapid, rule-based, procedural population of any arbitrarily sized part of the world with decorative content. 
+Depending on its configuration, a single Vegetation Layer can act as a small patch of flowers or intelligently blanket the entire world with a convincing variety and groupings of plants and objects.
+
+## Vegetation instances
+
+*Vegetation instances* are the objects placed throughout the world. They're created using vegetation descriptors. 
+They contain a unique ID, transform, other attributes, and a reference to the source vegetation descriptor.
+
+In O3DE Editor, vegetation instances appear within a vegetation area. They-re procedurally generated based on the vegetation behaviors that you can add to the vegetation area. 
+
+## Descriptors
+
+*Vegetation descriptors* are structures that specify all of the common details needed to represent a type of vegetation. It includes data for mesh and material assets, the type of vegetation render node that should be created, the affects of wind and physics, and many advanced parameters that can be enabled to override behavior of most of the included filters and modifiers. **Vegetation Descriptors** can be created in the **Vegetation Asset List** component or through the Asset Editor by creating and editing **Vegetation Descriptor List Assets**.
```

Comment:
"the affects of" -> "the effects of".

Reply:
Should this paragraph also be changed to remove mentions of "render nodes" and "wind and physics"? Since they are deprecated?

Reply:
Yep, absolutely, great catch! "render node" -> "instance", and remove "the effects of wind and physics", since those aren't specified here any more.